src/main/java/apoc/graph/util/FromConfig.java

```diff
@@ -0,0 +1,35 @@
+package apoc.graph.util;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static apoc.util.Util.toBoolean;
+
+public class FromConfig {
```

Comment:
what is a "From" Config?

Reply:
It's a configuration class that can be used in all the `apoc.graphs.from*` procedures maybe `GraphsFromConfig` it's a better name?